pkg/operator/ceph/cluster/watcher.go

```diff
@@ -516,7 +525,7 @@ func (c *clientCluster) createNetworkFence(ctx context.Context, pv corev1.Persis
 
 	networkFence := &addonsv1alpha1.NetworkFence{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      node.Name,
+			Name:      fenceResourceName(node.Name, driver),
```

Comment:
```suggestion
			Name:      fenceResourceName(node.Name, pv.Spec.CSI.Driver),
```

Reply:
Is that the full driver name though? We just want "rbd" or "cephfs"

Reply:
Yes, pv.Spec.CSI.Driver  contains full driver name, so won't be good to use that.